codemods/use-i18n-hook/example-before.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { translate, Text, Modal } from 'cozy-ui/transpiled/react'
+
+import OtherImports from 'other/imports'
+
+const ComponentUsingT = translate()(({ t }) => (
+  <Padded>
+    <Title>{t('Title1')}</Title>
+    <Text>{t('Text1')}</Text>
+  </Padded>
+))
+
+const ComponentUsingTAndF = ({ t, f }) => (
```

Comment:
```suggestion
const DumbComponentUsingTAndF = ({ t, f }) => (
```

Reply:
fix: Example had the wrong variable name